README.md

```diff
@@ -18,11 +18,19 @@ pip install -r requirements.txt
 
 Bases on [DID specification](https://www.w3.org/TR/did-core/#dfn-did-documents): "...DID documents contain information associated with a DID. They typically express verification methods, such as cryptographic public keys, and services relevant to interactions with the DID subject..."
 
-In context of Gaia-X, DID document contains at least one verification method to verify authorship of Gaia-X Credentials. scs-did-creator supports [RSA keys](https://en.wikipedia.org/wiki/RSA_(cryptosystem)) and [EC keys](https://en.wikipedia.org/wiki/Elliptic-curve_cryptography) as verification methods, only. If not already exist, create a public-private key pair with [OpenSSL](https://developers.yubico.com/PIV/Guides/Generating_keys_using_OpenSSL.html)
+There a several types of verification methods, such as [JsonWebKey2020 (JWK)](https://w3c-ccg.github.io/lds-jws2020/#json-web-key-2020 or [EcdsaSecp256k1VerificationKey2019](https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/). A complete list of supported types can be found in [DID Spec Registry](https://www.w3.org/TR/did-spec-registries/).
+
+**Note**: SCS scs-did-creator supports JWK, only.
+
+Public-Private JWK key pairs JWK could by different cryptographic algorithms.
```

Comment:
I don't understand this sentence. What is it supposed to mean?

Reply:
There is a verb missing. :-)

```suggestion
Public-Private JWK key pairs could be generates by different cryptographic algorithms.
```